tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -1785,6 +1785,15 @@ public function getEmailAddressForUser($userName) {
 	 * @return string
 	 */
 	public function getActualUsername($functionalUsername) {
+		$usernameFromEnv = \getenv('REPLACE_WITH_UNUSUAL_USERNAMES');
```

Comment:
```suggestion
		$usernameFromEnv = \getenv('REPLACE_USERNAMES');
```
Currently this is a boolean environment variable that "switches on" using replacement usernames. They might or might not be "unusual"

Reply:
fixed